main_routers/live2d_router.py

```
@@ -966,3 +966,110 @@ def open_model_directory(model_name: str):
+                    item_path = os.path.join(static_dir, item)
+                    item_real = os.path.realpath(item_path)
+                    if os.path.isdir(item_real) and model_real == item_real:
+                        marker_file = os.path.join(model_real, '.user_upload')
+                        if os.path.exists(marker_file):
+                            is_user_model = True
+                        break
+        except Exception:
+            pass
+        
+        if not is_user_model:
+            return JSONResponse(status_code=403, content={"success": False, "error": "只能删除用户导入的模型，无法删除内置模型"})
```

Ruff (0.14.10)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)